python/mxnet/optimizer.py

```diff
@@ -529,6 +529,55 @@ def update_multi_precision(self, index, weight, grad, state):
         self._update_impl(index, weight, grad, state,
                           multi_precision=use_multi_precision)
 
+
+@register
+class FTML(Optimizer):
```

Comment:
I thought we already have this

Reply:
We have FTRL (Follow the regularized leader). This PR adds FTML (Follow the moving leader).